### PR TITLE
Add more co founders

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #150

### CODE REVIEW

File: public/form.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
 var plusCircle = document.querySelector('#plus-circle');
-var coFounderInput = document.querySelector('#coFounder-name');
+var coFounderInput = document.querySelector('#co-founder');
+var childNumber = 0;
 plusCircle.addEventListener('click', function () {
-  coFounderInput.classList.toggle('dn');
+  var childNumber = document.getElementById('co-founder').childElementCount;
+    if(childNumber <= 4) {
+      var input = document.createElement('input');
```
Comment (Karyum) [inline public/form.js]: im not sure that var input is a good name for a variable

File: src/views/layouts/main.hbs [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" type="text/css" href="/style.css">
-    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="https://unpkg.com/tachyons/css/tachyons.min.css"/>
```
Comment (Karyum) [inline src/views/layouts/main.hbs]: why was this changed?

Comment (Suuuuha) [inline src/views/layouts/main.hbs]: Because I was getting a 503 error and it turns out because tachyons version is updated. removed the version for preventing fututre errors like that

**Comment by Suuuuha** [comment]
@Karyum changes made

**Comment by esraajb** [comment]
@Suuuuha so now how we get this data to be saved in the db?

**Comment by Suuuuha** [comment]
@esraajb it does save the data to the data base, and seperate the co-founders by commas when more than one co-founder added.
We just need to change the styling of the startup profile page so it would be compatible with multiple co founders

**Comment by esraajb** [comment]
@Suuuuha please raise an issue about the multiple co-founders problem.
